### ISSUE TITLE
Being able to select a range of tabs to move around or put into a subgroup.

### ISSUE THREAD

**Issue by jmcollis** [GarinZ/link-map#95]
Currently, I can't select a range of tabs to move or process at once. 

Being able to do this would help a lot. 

Running on latest Chrome on Mac.

**Comment by jasonhuang1115** [comment]
Yes, it will be great to be able to shift-slect multiple tabs to move or process at once.

**Comment by lyzzidc** [comment]
yes! I was just coming here to say this
